pkg/controller/pd_control.go

```diff
@@ -407,15 +396,24 @@ func (pc *pdClient) BeginEvictLeader(storeID uint64) error {
 	if res.StatusCode == http.StatusOK {
 		return nil
 	}
-	err2 := readErrorBody(res.Body)
-	if strings.Contains(err2.Error(), schedulerExisted) {
-		return nil
+
+	evictLeaderSchedulers, err := pc.GetEvictLeaderSchedulers()
```

Comment:
this should be exec before create evict leader, if the scheduler leader exists, then not need to post sheduler leader